client/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -1,38 +1,51 @@
 import Grid from '@material-ui/core/Grid';
 import CssBaseline from '@material-ui/core/CssBaseline';
-import CircularProgress from '@material-ui/core/CircularProgress';
+// import Box from '@material-ui/core/Box';
+// import AppBar from '@material-ui/core/AppBar';
+// import Toolbar from '@material-ui/core/Toolbar';
+// import CircularProgress from '@material-ui/core/CircularProgress';
 import useStyles from './useStyles';
-import { useAuth } from '../../context/useAuthContext';
-import { useSocket } from '../../context/useSocketContext';
-import { useHistory } from 'react-router-dom';
-import ChatSideBanner from '../../components/ChatSideBanner/ChatSideBanner';
-import { useEffect } from 'react';
+// import { useAuth } from '../../context/useAuthContext';
```

Comment:
bunch of commented out code here

Reply:
Yeah just so I could access the dashboard without worrying about auth.

Reply:
When pushing code to main we want to avoid commented out code. If it needs to be commented it can be removed :) We can always add it back in later if we need to.
In this case I would say you should uncomment `useAuth`. Remember that main is where all your teammates will also be pulling from, so we want to make sure it is the most up to date running version of our app!

Reply:
That makes sense. I'll fix it.